### PR TITLE
Added dropdowns to the menu

### PR DESCRIPTION
**Pull request by viv-codes**

### CODE REVIEW

**Comment by viv-codes** [comment]
image

**Review by galenguyer — APPROVED** [review]
LGTM

File: templates/navbar.html [hunk 1]
```diff
@@ -11,9 +11,17 @@
         <li class="nav-item my-auto">
           <a class="nav-link" href="https://csh.rit.edu">Public Site</a>
         </li>
+
+        <li class="nav-item dropdown" data-ng-repeat="section in sections">
+          <a class="nav-link dropdown-toggle" href="http://example.com" id="dropdown05" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">{{ section.name }}</a>
```
Comment (galenguyer) [inline templates/navbar.html:16]: Ack, this shouldn't be `example.com`

**Review by galenguyer — CHANGES_REQUESTED** [review]

File: templates/navbar.html (hunk 1 above)
Comment (viv-codes) [reply templates/navbar.html:16]: Will PR and fix immediately.

**Pull request merged**